File: src/go/fizzbuzz.go
package main

import "fmt"

func main() {
	for i := 1; i <= 100; i++ {
		if (i % 3 == 0 && i % 5 == 0) {
            fmt.Print("Fizzbuzz")
        } else if (i % 3 == 0) {
            fmt.Print("Fizz")
        } else if (i % 5 == 0) {
            fmt.Print("Buzz")
        } else {
            fmt.Print(i)
        }
		
		if (i % 10 == 0){
			fmt.Println()
		}else{
            fmt.Print(",")
        }
	}
}

File: src/rb/fizzbuzz.rb
1.upto(100) do |i|
    if i % 3 == 0 && i % 5 == 0
        print "FizzBuzz"
    elsif i % 3 == 0
        print "Fizz"
    elsif i % 5 == 0
        print "Buzz"
    else
        print i
    end

    if i % 10 == 0
        puts
    else
        print ","
    end
end